src/components/top-bar-equipment-seach-dialog/top-bar-equipment-search-dialog.tsx

```diff
@@ -146,11 +149,11 @@ export const TopBarEquipmentSearchDialog: FunctionComponent<
             }
             renderInput={(displayedValue, params) => (
                 <TopBarEquipmentSearchInput
-                    isSearchDisabled={disabledSearchReason !== ''}
                     displayedValue={displayedValue}
                     params={params}
                     setEquipmentType={setEquipmentTypeFilter}
                     equipmentType={equipmentTypeFilter}
+                    disabledSearchReason={disabledSearchReason}
```

Comment:
`disabledSearchReason` is already passed to the `displayedValue` by the `ElementSearchDialog` component

Reply:
Yees,indeed.